demos/modipulate-gml/test/main.c

```diff
@@ -0,0 +1,200 @@
+/* Demo test app for modipulate-gml */
+
+#include <stdio.h>
+#include "modipulate_gml.h"
+
+#ifdef _WIN32
+#include <windows.h>
```

Comment:
Definitely -- do we include windows.h anywhere else? Couldn't hurt to grep around a little